app/controllers/users_controller.rb

```diff
@@ -1,7 +1,65 @@
 class UsersController < ApplicationController
 
-  def list
+  before_action :find_user
```

Comment:
why not `before_action :find_user, only: [ .... ]`? This removes the need for the skip.